.gitattributes

```diff
@@ -1,8 +1,6 @@
-# Auto detect text files and perform LF normalization
-* text=auto
-# Declare files that will always have LF line endings on checkout.
-*.sh text eol=lf
-*.prop text eol=lf
-*.md text eol=lf
-*.xml text eol=lf
-META-INF/** text eol=lf
+# Declare files that will always have LF line endings on checkout.

```

Comment:
Do we still need this?

Reply:
yes. this is to prevent line endings from changing. if someone here uses windows to push and they dont change the line endings from crlf to lf in an xml or txt or sh file then it wont work on android or linux

Reply:
https://stackoverflow.com/questions/5834014/lf-will-be-replaced-by-crlf-in-git-what-is-that-and-is-it-important